tools/meson-check-api-docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eu
```

Comment:
I think it's best practice to have a separate `set -eu` rather than pass flags in the shebang. This way doing `sh -x tools/meson-check-api-docs.sh` doesn't loose the eu bits.

Reply:
hmm, if so we have to fix a number of scripts that way I figure

Reply:
I'll send a PR fixing this in scripts that have it this way.